Jenkinsfile

```diff
@@ -632,6 +638,67 @@ pipeline {
       }
     }
 
+    stage('Deploy Preview') {
+      when {
+        not {
+          branch 'PR-*'
+        }
+      }
+      steps {
+        setGitHubBuildStatus('nuxeo/preview', 'Deploy nuxeo preview', 'PENDING')
+        container('maven') {
+          dir('ci/helm/preview') {
+            echo """
+            ----------------------------------------
+            Deploy Preview environment
+            ----------------------------------------"""
+            // first substitute environment variables in chart values
+            sh """
+              mv values.yaml values.yaml.tosubst
+              envsubst < values.yaml.tosubst > values.yaml
+            """
+            // second create target namespace (if doesn't exist) and copy secrets to target namespace
+            script {
+              boolean nsExists = sh(returnStatus: true, script: "kubectl get namespace ${PREVIEW_NAMESPACE}") == 0
+              String noCommentOpt = '';
+              if (nsExists) {
+                noCommentOpt = '--no-comment'
+                // Previous preview deployment needs to be scaled to 0 to be replaced correctly
+                sh "kubectl -n ${PREVIEW_NAMESPACE} scale deployment preview --replicas=0"
+              } else {
+                sh "kubectl create namespace ${PREVIEW_NAMESPACE}"
+              }
+              // third build and deploy the chart
+              // To avoid jx gc cron job, reference branch previews are deployed by calling jx step helm install
+              // The -n name provided must be lowercase
```

Comment:
I think this comment would be more suitable just before the definition of `PREVIEW_NAMESPACE` where the lowercase is applied, or eventually before  `kubectl create namespace`, because it is really this instruction that requires lowercase.

Reply:
I put it at the declaration. This is the only place where I call toLowerCase() now so I think it belongs there to explain the call